src/casp/bq_scripts/anndata_to_avro.py

```diff
@@ -211,7 +281,12 @@ def default(self, o):
                 return float(o)
             if isinstance(o, np.ndarray):
                 return o.tolist()
-            return json.JSONEncoder.default(self, o)
+            else:
```

Comment:
Please move this class outside of `dump_ingest_info`, perhaps under `bq_scripts/encoders`.